contracts/argent_account.cairo

```diff
@@ -101,12 +120,80 @@ mod ArgentAccount {
         assert(signatures.len() == 4_u128, 'argent: signature format invalid');
         let signature_r = signatures.at(2_u128);
         let signature_s = signatures.at(3_u128);
-        let is_valid_guardian_signature = ecdsa::check_ecdsa_signature(
-            hash, guardian_, signature_r, signature_s
-        );
-        if is_valid_guardian_signature {
+        if ecdsa::check_ecdsa_signature(
+            message_hash, guardian_, signature_r, signature_s
+        ) {
             return true;
         }
-        ecdsa::check_ecdsa_signature(hash, guardian_backup::read(), signature_r, signature_s)
+        ecdsa::check_ecdsa_signature(
+            message_hash, guardian_backup::read(), signature_r, signature_s
+        )
+    }
```

Comment:
Pls revert all your changes to `isValidSignature`, `is_valid_signer_signature`, and `is_valid_guardian_signature`.

Remember to rebase your branch before pushing it so you're basing it on the latest changes